SAP/luigi
# Docu: add docu shell on direct loading of micro frontend

Redirect with docu shell when mf was directly opened
Keep bots like algolia without docu shell

Fixes #1232

How to test: 
Run develop mode and open the microfrontend directly: http://localhost:4001  .. it should go to localhost:4000 and keep the same url. 
Search bot exception test: Set your browser user-agent to DocSearch and it should stay on 4001
